src-graphql/images-query.js

```diff
@@ -0,0 +1,9 @@
+export default function imagesQuery(fields = ['id', 'src', 'altText']) {
```

Comment:
It may actually be better to be specific and use the graphql terms then. Where it's a paginated set, maybe call it an `imageConnectionQuery`, and when it's a singular image, call it `imageQuery`. This approach would mean you should rename some of your other queries too though.